src/components/StateFooter/index.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import { Button, Grid, makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+    maxWidth: 1000
+  },
+  button: {
+    textTransform: "none"
+  }
```

Comment:
Why did you need this for the button?

Reply:
The button text defaults to all caps, so removed with `textTransform`

Reply:
I see. That's actually a good idea. Better yet, why don't you place this property in the global `src/theme/index.ts theme file. That way all buttons will get this. What do you think? Place the exact same object inside the typography object 
```suggestion
  .. // other theme properties
  typography: {
    ... 
    button: {
      textTransform: "none"
    }
  }
```